### ISSUE TITLE
QR kódy

### ISSUE THREAD

**Issue by Pulcik** [Clevis/ReKola-Public#38]
Co takhle zapojit QR kódy? 
Asi by bylo třeba vyzkoušet optimální velikost a ergonomicky vhodné místo pro QR kód.

**Comment by Piskvor** [comment]
No, teď jsem si půjčoval kolo, a byl jsem rád za lidsky čitelný kód - představa, že bych se v této velikosti/umístění snažil vyfotit QR kód, mě poněkud děsí (řečené kolo - Mlada - nemá v tuto chvíli lepších ploch). Pokud by to byl doplněk k viditelnému číselnému kódu, proč ne - to by se konečně dalo použít i k propagaci, žejo (nepřihlášení dostanou stránku "co jsou ReKola") - ale nevím, jak moc by to bylo implementačně složitý.

**Comment by Pulcik** [comment]
Na čitelných číslech se pracuje a přesně tak jsem to i mínil. Byl by to doplněk k číslu.

**Comment by Piskvor** [comment]
(jenom upřesním: proti současné čitelnosti čísel nic nemám, připadá mi dostačující; QR kód o podobných rozměrech by medle čitelný nebyl)

**Comment by HonzaaD** [comment]
omlouvam se, otevrel jsem tuto otazku dnes znovu https://github.com/Clevis/ReKola-Public/issues/105 myslim, ze by to bylo fajn.. Zkousel jsem dnes precist QR kod na kole, kde je http://www.protect6.cz a zadny problem..
